src/topojson/topojson_intf.ml

```diff
@@ -282,6 +282,21 @@ module type S = sig
 
     val to_json : ?bbox:float array -> t -> json
     val of_json : json -> (t, [ `Msg of string ]) result
+
+    val extract_lines :
```

Comment:
We won't actually ever expose these internal functions. Instead, eventually, we will just have something like:

```ocaml
val geojson_to_topojson : Geojson.t -> Topojson.t
```